docs/app/pages/components/components-sections/input-controls/radio-button/radio-button.component.html

```diff
@@ -75,7 +72,20 @@
 </uxd-api-properties>
 
 <uxd-api-properties tableTitle="Outputs">
-    <tr uxd-api-property name="valueChange" type="any" defaultValue="">
+    <tr uxd-api-property name="valueChange" type="any">
+        Emits when the value has been changed.
+    </tr>
+</uxd-api-properties>
+
+<uxd-api-properties tableTitle="uxRadioButtonGroup Inputs">
+    <tr uxd-api-property name="value" type="any">
+        This should be a two way binding and will store the currently selected option. Each radio button in the same
```

Comment:
"Each radio button in the same  group should have the same <code>value</code> variable." - This is not the case, the group negates the need for each radio button to have its only value binding.

Reply:
Changed wording